cmd/snap/cmd_snap_op.go

```diff
@@ -115,7 +115,11 @@ func (x *cmdRemove) Execute([]string) error {
 		return err
 	}
 
-	return wait(cli, changeID)
+	if err := wait(cli, changeID); err != nil {
+		return err
+	}
+	fmt.Println("Done")
```

Comment:
Actually, we can probably drop this line now that we agreed to have the list output.